tests/Fixtures/Issue/Issue198Owner.php

```diff
@@ -7,44 +8,47 @@
 /**
  * @ORM\Entity()
  */
-class Issue198Owner
+final class Issue198Owner
 {
-    /** @ORM\Id @ORM\Column(type="integer") @ORM\GeneratedValue(strategy="AUTO") */
-    protected $id;
-    
+    /**
+     * @ORM\Id @ORM\Column(type="integer")
+     * @ORM\GeneratedValue(strategy="AUTO")
+     */
+    private $id;
+
     /**
      * @ORM\OneToMany(targetEntity="Issue198Car", mappedBy="owner")
      */
-    protected $cars;
-    
+    private $cars;
+
     public function __construct()
     {
         $this->cars = new ArrayCollection();
     }
-    
+
     public function setId($id)
     {
         $this->id = $id;
     }
-    
+
     public function getId()
     {
         return $this->id;
     }
-    
+
     public function addCar(Issue198Car $car)
     {
         if (!$this->cars->contains($car)) {
             $car->setOwner($this);
             $this->cars[] = $car;
         }
     }
-    
+
     public function removeCar(Issue198Car $car)
     {
         $this->cars->removeElement($car);
     }
-    
```

Comment:
Why this was not fix by the linter ?

The lint action was green, and the rules
```
no_trailing_whitespace
no_whitespace_in_blank_line
```
are included in the Symfony ruleset.

Reply:
Oh, it's in the Fixture folder

Reply:
I made https://github.com/sonata-project/dev-kit/pull/1418